src/metadata.yaml

```diff
@@ -1,27 +1,28 @@
-name: "keepalived"
-summary: "Failover and monitoring daemon for LVS clusters"
+name: keepalived
+display-name: Keepalived
+summary: Failover and monitoring daemon for LVS clusters
 maintainers:
-  - "Konstantinos Tsakalozos <kos.tsakalozos@canonical.com>"
-  - "Valentin Boucher <valentin.boucher@kontron.com>"
+- "Konstantinos Tsakalozos <kos.tsakalozos@canonical.com>"
+- "Valentin Boucher <valentin.boucher@kontron.com>"
 description: |
   keepalived is used for monitoring real servers within a Linux Virtual
   Server (LVS) cluster.
 docs: https://discourse.charmhub.io/t/keepalived-docs-index/6043
 tags:
-  - "networking"
-series:
-  - noble
-  - jammy
-  - focal
+- networking
+
+assumes:
+- juju >= 3.4
```

Comment:
Required b/c the addition of noble forces the charm to be used on juju 3.4 and beyond.  No prior juju client will support the inclusion of noble as a potential base